docs/docsite/rst/intro_inventory.rst

```diff
@@ -82,10 +82,11 @@ In YAML:
 
 .. code-block:: yaml
 
-    hosts:
-      jumper:
-        ansible_port: 5555
-        ansible_host: 192.0.2.50
+    all:
```

Comment:
this is not an example of a full inventory file, that is already done above, this is just an extract of one.

Reply:
I would have no problem if it had an elipsis (...) above the `hosts:` line to indicate that it was an extract.

However, every other inventory on this page (both INI and YAML) is valid as a standalone inventory, and I found it confusing that one was not.

This page provides no formal definition of the YAML inventory format, and it doesn't actually say that the top-level key must be a group name; you are supposed to pick it up by example.  The first example ("A YAML version would look like:") does include a top level `all:` key, but it is followed shortly afterwards by the `jumper:` example which does not.

Reply:
im fine with adding a  `...` ,  most of the 'ini' examples are valid as full inventories as it does not really require a hierarchy as the YAML one does. This is the intro document so I did not think a full spec should be here (but we SHOULD link to one) ... and that full spec also does not exist for the ini one (yaml one is  'documented' in examples/hosts.yaml).

Reply:
also, in newer version this will be in each plugin's description: http://docs.ansible.com/ansible/devel/plugins/inventory/yaml.html